indico/modules/events/management/templates/clone_event.html

```diff
@@ -0,0 +1,107 @@
+{% from 'forms/_form.html' import form_header, form_fieldset, form_footer, form_row, form_rows %}
+{% from 'message_box.html' import message_box %}
+
+{% macro invisible_field(field) -%}
+    {% set field_value = field._value() if field._value else field.data %}
+    {% set data = [field_value] if field_value is string else ([field_value|tojson] if field_value is mapping else field_value) %}
+    {% for value in data %}
+        <input type="hidden" name="{{ field.name }}" value="{{ value | forceescape }}">
+    {% endfor %}
+{%- endmacro %}
+
+{% macro invisible_fields(form, skip) %}
+    {% for field in form %}
+        {% if field.id not in skip %}
+            {{ invisible_field(field) }}
+        {% endif %}
+    {% endfor %}
+{% endmacro %}
+
+{% block content %}
+    <div id="event-clone-form-container"
+         data-preview-url="{{ url_for('event_management.clone_preview', event) }}"
```

Comment:
But isn't it loaded in an AJAX dialog? So it'll still be inside the management blueprint